legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/api/GitlabWorkflowApi.java

```diff
@@ -178,22 +179,18 @@ public Workflow getWorkflow(String workflowId)
             {
                 boolean limited = (limit != null) && (limit > 0);
                 int itemsPerPage = limited ? Math.min(limit, ITEMS_PER_PAGE) : ITEMS_PER_PAGE;
-                PipelineApi pipelineApi = getGitLabApi(this.projectId.getGitLabMode()).getPipelineApi();
+                PipelineApi pipelineApi = getGitLabApi().getPipelineApi();
                 Pager<Pipeline> pager = withRetries(() -> pipelineApi.getPipelines(this.projectId.getGitLabId(), null, null, getRef(), false, null, null, null, null, itemsPerPage));
                 Stream<Pipeline> pipelineStream = PagerTools.stream(pager);
-                Set<String> revisionIdSet = (revisionIds == null)
-                        ? Collections.emptySet()
-                        : StreamSupport.stream(revisionIds.spliterator(), false)
-                        // make sure various synonymous aliases are grouped together properly
-                        .filter(Objects::nonNull).map(revisionId ->
-                        {
-                            RevisionAlias alias = getRevisionAlias(revisionId);
-                            return (alias == RevisionAlias.REVISION_ID) ? revisionId : alias.getValue();
-                        })
-                        .distinct()
-                        // resolve revision alias when possible
-                        .map(revisionId -> resolveRevisionId(revisionId, getRevisionAccessContext()))
-                        .collect(Collectors.toSet());
+                Set<String> revisionIdSet = (revisionIds == null) ? Collections.emptySet() : StreamSupport.stream(revisionIds.spliterator(), false)
+                    // make sure various synonymous aliases are grouped together properly
+                    .filter(Objects::nonNull).map(revisionId ->
+                    {
+                        RevisionAlias alias = getRevisionAlias(revisionId);
+                        return (alias == RevisionAlias.REVISION_ID) ? revisionId : alias.getValue();
+                    }).distinct()
+                    // resolve revision alias when possible
+                    .map(revisionId -> resolveRevisionId(revisionId, getRevisionAccessContext())).collect(Collectors.toSet());
```

Comment:
This is a weird formatting mix. Of course, so was the previous formatting. Some stream operations get their own line, others are grouped on a single line. For long chains of stream operations, I think it's clearest to have each operation on a separate line.